parm/data_locations.yml

```diff
@@ -236,6 +236,20 @@ RAP:
     file_names:
       <<: *rap_file_names
 
+RRFS:
+  hpss:
+    protocol: htar
+    file_names: &rrfs_file_names
+      anl:
+        - rrfs.t{hh}z.prslev.f{fcst_hr:03d}.conus.grib2
+      fcst:
+        - rrfs.t{hh}z.prslev.f{fcst_hr:03d}.conus.grib2
```

Comment:
Should these be `rrfs.t{hh}z.prslev.f{fcst_hr:03d}.grib2` rather than `rrfs.t{hh}z.prslev.f{fcst_hr:03d}.conus.grib2`?

Attempting to run the new test on Hera results in `get_extrn_ics` and `get_extrn_lbcs` failing because there are no files named `rrfs.t{hh}z.prslev.f{fcst_hr:03d}.conus.grib2`, but there are files named `rrfs.t{hh}z.prslev.f{fcst_hr:03d}.grib2`.

Making this change allows the test to run successfully on Hera.

Reply:
@MichaelLueken - the files `rrfs.t{hh}z.prslev.f{fcst_hr:03d}.grib2` are for full RRFS files. 
Somehow the conus files were removed from Hera staging location when I started working with the full RRFS files. Now the `rrfs.t{hh}z.prslev.f{fcst_hr:03d}.conus.grib2` are placed back, and that's what is needed for the current stage of the RRFS work integration.